src/main/java/com/klodnicki/Bike/v2/service/ChargingStationService.java

```diff
@@ -1,7 +1,44 @@
 package com.klodnicki.Bike.v2.service;
 
+import com.klodnicki.Bike.v2.DTO.station.StationForAdminResponseDTO;
+import com.klodnicki.Bike.v2.model.entity.ChargingStation;
+import com.klodnicki.Bike.v2.repository.ChargingStationRepository;
+import org.modelmapper.ModelMapper;
 import org.springframework.stereotype.Service;
 
+import java.util.ArrayList;
+import java.util.List;
+
 @Service
 public class ChargingStationService {
+
+    private final ChargingStationRepository chargingStationRepository;
+
+    ModelMapper modelMapper = new ModelMapper();
```

Comment:
This field should not look like that. I noticed there is the same line also in other services. What's the reason that it is without accessor? This would be much better if you changed it to:
```suggestion
    private final ModelMapper modelMapper;
```
and used dependency injection mechanism. You would need to create a bean from modelmapper.

Reply:
you are absolutely right! I used @Autowired annotation to inject dependency and also I created config class for ModelMapper to define a bean

Reply:
Defining bean is a good choice, but using @Autowired isn't.

Why to not use this annotation:
![image](https://github.com/BartoliniAndBorderCollies/Bike.v2_REST_API/assets/33954943/04e62ace-b419-4613-bb21-8e4f52bc7d58)

Reply:
I tested it and it works properly, I understand that problem may arise in the future therefore I change it from @Autowired to constructor injection